### PR TITLE
Change cnf pp to print the quadruplet

### PR DESCRIPTION
**Pull request by GabinDDL**

### CODE REVIEW

File: lib/cnf_graph.ml [hunk 1]
```diff
@@ -49,6 +39,20 @@ let get_variable_value var dim =
   let x = x mod dim in
   (not, x, y, t, c)
 
+let pp_var dim fmt var =
```
Comment (ThomArr) [inline lib/cnf_graph.ml:42]: The problem is that the coef of the base is not the dimension of the grid. It depends on the large value between:
-The number of colours
-The width of the grid
-Height of grid
-Maximum time

**Review by ThomArr — APPROVED** [review]

File: lib/cnf_graph.ml (hunk 1 above)
Comment (GabinDDL) [reply lib/cnf_graph.ml:42]: I don't understand : I use exactly the same variable `dim` you use to call `get_variable_value`, which is what I need
If It's wrong, all the functions that use `dim` are wrong too no ?

Comment (ThomArr) [reply lib/cnf_graph.ml:42]: yes they are wrong 😔... I edited this in branch 2-CNF

Comment (GabinDDL) [reply lib/cnf_graph.ml]: ok it's now fixed

**Review by ThomArr — APPROVED** [review]

**Pull request merged**